src/lib/common/selector.py

```diff
@@ -51,18 +60,18 @@ def index(self, config):
         NOTE: should be a relatively light pass that designates the space to be retrieved.
         No options for parallelisation, run on a single CPU.
         """
-        return NotImplemented
+        return pd.DataFrame()
```

Comment:
This method should still raise a `NotImplemented` error here, as mtriage should not allow a selector to be constructed without the method.

Reply:
Fixed in latest commits